modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
+
+Garbage Collection Time::
+The approximate accumulated collection elapsed time in milliseconds, grouped by type of response in automatic intervals.
+
+Par New Collection Count::
+The number of collections for Parnew GC, grouped by type of response in automatic intervals.
+
+Par New Collection Time::
+The approximate accumulated Parnew collection elapsed time in milliseconds, grouped by type of response in automatic intervals.
+
+Classes Loaded::
+The number of currently loaded classes, grouped by type of response in automatic intervals.
+
+Classes Loaded Total::
+The number of classes loaded since JVM start, grouped by type of response in automatic intervals.
+
+Classes Unloaded::
+The number of classes unloaded since JVM start, grouped by type of response in automatic intervals.
+
+Heap Committed::
+The amount of heap memory committed by the JVM, grouped by type of response in automatic intervals.
+
+Heap Used::
+The amount of heap memory utilized by the JVM, grouped by type of response in automatic intervals.
+
+Thread Count::
+The number of simultaneous requests.
+
+JVM Uptime::
+The time since JVM started, grouped by type of response in automatic intervals.
+
+Par Eden graphs (Used, Max, Init, and Committed)::
+The usage metrics for “Eden Space” JVM Memory, grouped by type of response in automatic intervals.
+
+Par Survivor graphs (Used, Max, Init, and Committed)::
+The usage metrics for the Par Eden Space pool.
+
+Metaspace graphs (Used, Max, Init, Committed)::
+The usage metrics for Java memory space.
+
+Code Cache graphs (Used, Max, Init, Committed)::
+The usage metrics for the Code Cache pool.
+
+Compressed Class Space graphs (Used, Max, Init, Committed)::
+The usage metrics for values of the metadata for compressed class pointers.
+
+Committed Virtual Memory::
+The amount of virtual memory guaranteed to be available to the running process.
+
+[[infrastructure-charts]]
+== Infrastructure
+
+The following charts appear on the *Infrastructure* dashboard:
+
+Application Process CPU Load::
+The amount of CPU load the JVM uses.
+
+JVM CPU Time::
+The cumulative CPU time consumed by the JVM.
+
+Operating System Swap Space::
+The amount of memory the system is swapping, grouped by worker in automatic intervals.
+
+CPU Utilization %::
+The percentage of CPU used by each worker over time.
+
+Memory Utilization::
+The amount of memory used by each worker over time.
+
+Total System Processors::
+The number of system processors for workers available over time.
+
+Total System Memory::
+The amount of system memory available for workers over time. If you have a Titanium subscription and the selected application is hybrid, this graph changes to *System Memory*.
+
+System Memory::
+The total free system memory per worker. This graph appears only if you have a Titanium subscription and the selected application is hybrid.
+
+Thread Count - Server::
+The number of simultaneous requests for a worker over time.
```

Comment:
The average number of threads of the JVM, grouped by worker in automatic intervals. The JVM used in this chart is the Mule runtime JVM process.

<< copy from Overview for the same chart >>